package.json

```diff
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/ethereumjs/ethereumjs-blockchain#readme",
   "dependencies": {
+    "@babel/runtime": "^7.3.1",
```

Comment:
I think ``@babel/runtime`` can be moved to the ``devDependencies`` section since we are building ``dist/index.js`` prior to an ``npm publish``

Reply:
Babel [docs](https://babeljs.io/docs/en/babel-plugin-transform-runtime) say:

> The transformation plugin is typically used only in development, but the runtime itself will be depended on by your deployed code. See the examples below for more details.

This is consistent with my experience, when I injected the newly built `ethereumjs-blockchain` in VM, there were errors that `regeneratorRuntime is not defined` which were fixed after I installed `@babel/runtime` in _VM_.

Reply:
@s1na Yes, you're right!  My apologies 🤦🏻‍♂️ 

However, I didn't have any issues when I ran the ``ethereumjs-vm`` tests with the injected ``ethereumjs-blockchain#refactor`` library.

Can you tell me how exactly you injected the babel-ified blockchain library into the VM? And what was the command you used to generate the ``regeneratorRuntime`` error?

Here is what I did (using ``node 8.15.0`` and ``npm 6.4.1``):

```
mkdir tmp && cd tmp
git clone -b refactor https://github.com/ethereumjs/ethereumjs-blockchain
git clone https://github.com/ethereumjs/ethereumjs-vm
cd ethereumjs-blockchain && npm i && npm run build && cd ..
cd ethereumjs-vm
npm r ethereumjs-blockchain
npm i ../ethereumjs-blockchain
npm run testBlockchain
```

This did not generate a ``regeneratorRuntime`` error for me.

Reply:
@s1na Can you reply here?

Reply:
Sorry for the delay.

I injected it by doing an `npm pack` and copying the tarball to `ethereumjs-vm/node_modules`.

I think the reason you didn't get an error might be because `@babel/runtime` is in the dependencies in the `refactor` branch and it was installed when you did `npm i ../ethereumjs-blockchain`.

I just tried your script, making sure to remove `@babel/runtime` from dependencies immediately after cloning and I got an error.

Reply:
Should be safe/sufficiently validated then to leave it in the dependencies, also considering that babel is stating the necessity in the docs.